src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -730,8 +734,7 @@ await ReadCharAsync(cancellationToken).ConfigureAwait(false) != '1')
             foreach (KeyValuePair<string, IEnumerable<string>> header in headers)
```

Comment:
I like `var`, but only when it's quick and easy for me to determine while reading the code what type the variable is.  It's for reasons like that that we use `var` but only when the type is obvious from the RHS, e.g. `new`, casts, etc.